src/features/pass-survey/lib/types/payload.ts

```diff
@@ -351,14 +374,20 @@ export type RadioResponse = string;
 
 export type TimeResponse = HourMinute;
 
-export type StackedCheckboxResponse =
-  | {
-      rowId: string;
-      optionIds: string[];
-    }[]
-  | null;
+export type StackedCheckboxResponse = {
+  rowId: string;
+  optionIds: string[];
+}[];
```

Comment:
The current answer DTO:
https://mindlogger.atlassian.net/wiki/spaces/MINDLOGGER1/pages/214499343/Slider+Rows+Answer

Please correct